core/rpc/src/Mu/Rpc.hs

```diff
@@ -27,66 +28,82 @@ import qualified Language.Haskell.TH as TH
 import           Mu.Schema
 import           Mu.Schema.Registry
 
+-- | Packages whose names are given by type-level strings.
+type Package' = Package Symbol Symbol
 -- | Services whose names are given by type-level strings.
 type Service' = Service Symbol Symbol
 -- | Annotations for services. At this moment, such
 --   annotations can be of any type.
 type ServiceAnnotation = Type
 
+-- | A package is a set of services.
+data Package serviceName methodName
+  = Package (Maybe serviceName)
```

Comment:
Why packages "maybe" have a name? 🤔

Reply:
Either `language-avro` or `language-protobuf` have a `Maybe Name`, so I changed it here as a consequence.